<launch>
  <rosparam command="load" 
    file="$(find iai_boxy_controller_configuration)/config/boxy_arm_pos_controllers.yaml"/>

  <node name="r_arm_pos_controller" pkg="iai_control_utils" type="joint_p_ctrl.py" args="" output="screen">
    <remap from="~pos_command_in" to="~command"/>
    <remap from="~vel_command_out" to="/r_arm_vel/command_raw"/>
    <remap from="~joint_states" to="/r_arm_vel/joint_states"/>
  </node>

</launch>
